src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,24 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    public static final User[] users = new User[]{
```

Comment:
Please, don't change the method signature
```suggestion
    private static final User[] users = new User[] {
```